src/App.vue

```diff
@@ -41,7 +42,11 @@
       },
       request(url) {
         fetch(url, { headers: { Accept: "application/json" } })
-          .then((response) => this.response = response);
+          .then((response) => this.response = response)
+          .catch(function(error) {
```

Comment:
You should prefer the `=>` notation for functions unless you have a good reason not to.

Reply:
Done